<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="CD059" name="tc_BasicDispatch_SelectPrinter_SampleAcknowledgementByTechnician_DDCRefundCancel.S45_Canceling2of4ServicesWithBothBillAndItemLevelDiscountInPercentageAndPatientNotPaidTheBill" tests="1" failures="0" timestamp="11 Jun 2018 07:33:20 GMT" time="387.161" errors="1">
  <testcase name="s45_Canceling2of4ServicesWithBothBillAndItemLevelDiscountInPercentageAndPatientNotPaidTheBill" time="387.161" classname="tc_BasicDispatch_SelectPrinter_SampleAcknowledgementByTechnician_DDCRefundCancel.S45_Canceling2of4ServicesWithBothBillAndItemLevelDiscountInPercentageAndPatientNotPaidTheBill">
    <error type="java.lang.Exception" message="Patient has due but alert not showing as &#039;Patient has dues&#039; when selecting the patient in despatch screen">
      <![CDATA[java.lang.Exception: Patient has due but alert not showing as 'Patient has dues' when selecting the patient in despatch screen
	at tc_BasicDispatch_SelectPrinter_SampleAcknowledgementByTechnician_DDCRefundCancel.S45_Canceling2of4ServicesWithBothBillAndItemLevelDiscountInPercentageAndPatientNotPaidTheBill.s45_Canceling2of4ServicesWithBothBillAndItemLevelDiscountInPercentageAndPatientNotPaidTheBill(S45_Canceling2of4ServicesWithBothBillAndItemLevelDiscountInPercentageAndPatientNotPaidTheBill.java:254)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.TestNG.privateMain(TestNG.java:1442)
	at org.testng.TestNG.main(TestNG.java:1411)
Caused by: org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of element located by By.xpath: .//*[@id='modernAlert']/p (tried for 10 second(s) with 500 MILLISECONDS interval)
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:80)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:232)
	at pageObjects.PO_MasterControl.getAlertMsg(PO_MasterControl.java:243)
	at tc_BasicDispatch_SelectPrinter_SampleAcknowledgementByTechnician_DDCRefundCancel.S45_Canceling2of4ServicesWithBothBillAndItemLevelDiscountInPercentageAndPatientNotPaidTheBill.s45_Canceling2of4ServicesWithBothBillAndItemLevelDiscountInPercentageAndPatientNotPaidTheBill(S45_Canceling2of4ServicesWithBothBillAndItemLevelDiscountInPercentageAndPatientNotPaidTheBill.java:248)
	... 24 more
Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate an element using By.xpath: .//*[@id='modernAlert']/p
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'CD059', ip: '192.168.1.56', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_131'
Driver info: driver.version: unknown
	at org.openqa.selenium.support.ui.ExpectedConditions.lambda$findElement$0(ExpectedConditions.java:883)
	at java.util.Optional.orElseThrow(Unknown Source)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:882)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:44)
	at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:206)
	at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:202)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:209)
	... 26 more
]]>
    </error>
  </testcase> <!-- s45_Canceling2of4ServicesWithBothBillAndItemLevelDiscountInPercentageAndPatientNotPaidTheBill -->
</testsuite> <!-- tc_BasicDispatch_SelectPrinter_SampleAcknowledgementByTechnician_DDCRefundCancel.S45_Canceling2of4ServicesWithBothBillAndItemLevelDiscountInPercentageAndPatientNotPaidTheBill -->
